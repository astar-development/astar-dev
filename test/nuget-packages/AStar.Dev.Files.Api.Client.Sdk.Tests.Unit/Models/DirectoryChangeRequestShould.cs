using AStar.Dev.Files.Api.Client.SDK.Models;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

public sealed class DirectoryChangeRequestShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new DirectoryChangeRequest { FileName = "MockFileName", NewDirectoryName = "NewMockDirectoryName", OldDirectoryName = "MockOldDirectoryName" }.ToString().ShouldMatchApproved();
}
