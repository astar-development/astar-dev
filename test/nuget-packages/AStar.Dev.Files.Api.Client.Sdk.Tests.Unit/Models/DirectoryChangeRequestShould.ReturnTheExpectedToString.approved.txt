{
  "oldDirectoryName": "MockOldDirectoryName",
  "newDirectoryName": "NewMockDirectoryName",
  "fileName": "MockFileName"
}